CHANGES.rst

```diff
@@ -82,6 +82,8 @@ Specviz2d
 API Changes
 -----------
 
+- Plugin Table components now support row selection. [#2856]
+
```

Comment:
since this isn't actually used anywhere, it might not need a changelog entry?

Reply:
Entirely up to y'all, just need the instruction and I can remove it!

Reply:
Since people can use this downstream, let's keep it.